source/documentation/runbooks/index.html.md.erb

```diff
@@ -0,0 +1,56 @@
+---
+title: Runbooks
+---
+
+# <%= current_page.data.title %>
```

Comment:
if it helps, heres most of the runbooks -

- [digideps](https://github.com/ministryofjustice/opg-digideps/tree/main/docs/runbooks)
- [i & p](https://github.com/ministryofjustice/opg-data-lpa-instructions-preferences/tree/main/docs/runbooks)
- [make](https://github.com/ministryofjustice/opg-lpa/tree/main/docs/runbooks)
- [serve](https://github.com/ministryofjustice/serve-opg/tree/main/docs/runbooks)
- [use](https://github.com/ministryofjustice/opg-use-an-lpa/tree/main/docs/runbooks)